revel/skeleton/conf/app.conf.template

```diff
@@ -51,8 +51,10 @@ cookie.prefix = REVEL
 # server. This makes the cookie less likely to be exposed to cookie theft via
 # eavesdropping.
 #
-# In dev mode, this will default to false, otherwise it will
-# default to true.
+# Defaults to false. If 'http.ssl' is enabled, this will automatically be enabled.
```

Comment:
not `this will automatically be enabled ` should be `this will be defaulted to true` The app.conf file value should still have the over-ride on this